.github/workflows/int_test.yml

```diff
@@ -57,7 +62,7 @@ permissions:
   id-token: 'write'
 
 env:
-  KV_NAME: "opskv${{ github.run_number }}"
+  KV_NAME: "opskv${{ github.run_number }}int"
```

Comment:
ok - will look into other identifiers too

Reply:
We decided to:
1. use `opsint${{ github.run_id }}` so we get out of clashing (and hopefully avoid clashing later since run_id should be unique per repo.
2. change the clean-up pipeline to purge all keyvaults + remove anything `az iot ops delete` did not take care of.